terraform/interpolate.go

```diff
@@ -519,6 +519,10 @@ func (i *Interpolater) interpolateListAttribute(
 	log.Printf("[DEBUG] Interpolating computed list attribute %s (%s)",
 		resourceID, attr)
 
+	if attr == config.UnknownVariableValue {
```

Comment:
Would a comment help here? Maybe slightly rhetorical since I'm unsure at first glance as an outsider to this PR why this behavior needs to be here. At least covered by tests! :D